controller/src/beerocks/master/db/db.h

```diff
@@ -59,6 +59,7 @@ class db {
         bool load_front_measurements;
         bool load_health_check;
         bool load_monitor_on_vaps;
+        bool certification_mode;
```

Comment:
I'm not sure what the difference is between sDbMasterConfig and sDBMasterSettings, but I feel it is more appropriate to put this in Settings, together with things like health_check and rdkb_extensions.

Reply:
sDBMasterSettings if for dynamic configuration which we do logic 'AND' with the configuration in sDbMasterConfig.
Since we don't do it and override the configuration in the config file, it is defined in sDBMasterSettings.